core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -1034,4 +1035,13 @@ default Stream<HostingGoalForTablet> getTabletHostingGoal(final String tableName
     throw new UnsupportedOperationException();
   }
 
+  /**
+   * @return a stream of tablets information for tablets that fall in the specified range
+   * @since 4.0.0
+   */
+  default Stream<TabletInformationImpl> getTabletInformation(final String tableName,
```

Comment:
This should return  the public API type

```suggestion
  default Stream<TabletInformation> getTabletInformation(final String tableName,
```

Reply:
@keith-turner I took your suggestion and then found that it could be simplified further by adding a cast within the lambda. I thought I had tried that before, but I must have been mistaken.

Once this PR is merged I will address the getTabletHostingGoal method that you mentioned above.